articles/quickstart/spa/_includes/_calling_api_preamble.md

```diff
@@ -1,3 +1,7 @@
-It's likely that your single page app will need to consume resources from a data API. Since you are adding authentication to your app, presumably you need to limit access to these resources such that they are only accessible by an authenticated user who is authorized to retrieve them. Auth0 provides [API Authorization](https://auth0.com/docs/api-auth) to accomplish this.
+Some single-page apps use resources from data APIs. You may want to restrict access to those resources, so that only authenticated users with sufficient privileges can access them. Auth0 lets you manage access to these resources using [API Authorization](https://auth0.com/docs/api-auth).
 
-This tutorial will guide you through how to make calls from your application to your data API to access protected resources. It does not, however, cover how to add protection to your API itself. For instructions on protecting your API, please follow the [Backend/API quickstart documentation](quickstart/backend) specific to your situation.
\ No newline at end of file
+This tutorial shows you how to access protected resources in your data API. 
```

Comment:
I haven't read the complete doc from start to finish, just fragments, but I don't get the `data API`. Why not `This tutorial shows you how to access protected resources in your API.`?